pytensor/link/pytorch/dispatch/scalar.py

```diff
@@ -0,0 +1,328 @@
+import functools
+import typing
+from collections.abc import Callable
+
+import torch
+
+from pytensor.link.pytorch.dispatch.basic import pytorch_funcify
+from pytensor.scalar import Softplus
+from pytensor.scalar.basic import (
+    Add,
+    Cast,
+    Clip,
+    Composite,
+    Identity,
+    IntDiv,
+    Mod,
+    Mul,
+    ScalarOp,
+    Second,
+    Sub,
+)
+from pytensor.scalar.math import (
+    BetaIncInv,
+    Erf,
+    Erfc,
+    Erfcinv,
+    Erfcx,
+    Erfinv,
+    GammaIncCInv,
+    GammaIncInv,
+    Iv,
+    Ive,
+    Log1mexp,
+    Psi,
+    TriGamma,
+)
+
+
+def try_import_tfp_jax_op(op: ScalarOp, jax_op_name: str | None = None) -> Callable:
+    try:
+        import tensorflow_probability.substrates.jax.math as tfp_jax_math
```

Comment:
I guess that needs some more adapting, I don't think we want to use any jax code here...

Reply:
Will `tensorflow_probability` also be required?
It does have some functions that are not available in pytorch (For example: `tfp.math.betainc`)

Reply:
I don't think tensorflow_probability generates any pytorch code, so it's not relevant here?

Reply:
@HarshvirSandhu `tfp` only provides JAX which we can't use here.

Reply:
How would we implement the Ops (like `GammaIncInv`) that don't have a direct implementation in pytorch? Will we use scipy?

Reply:
3 options:
1. Don't implement it
2. Find a PyTorch implementation that does so
3. Code on ourselves